code/game/objects/items/stacks/medical.dm

```diff
@@ -127,6 +127,23 @@
 	self_delay = 20
 	max_amount = 12
 
+/obj/item/stack/medical/gauze/wirecutter_act(mob/living/user, obj/item/I)
+	if(get_amount() < 2)
+		to_chat(user, "<span class='warning'>You need at least two gauze to do this!</span>")
+		return
+
+	else

```

Comment:
Doesn't need to be both an early-`return` and an `else`, I suggest removing the `else`.